lib/rubocop/cop/rspec/rails/http_status.rb

```diff
@@ -41,12 +54,13 @@ class HttpStatus < Base
           PATTERN
 
           def on_send(node)
```

Comment:
It seems this cop has a bias towards the new style, as it inspects `have_http_status` calls, but doesn't care if `be_*` are used when configured to use `numeric` or `symbolic` styles.

It's not a big problem, since otherwise it would detect false positives in e.g. non-controller/request specs.

Maybe it makes sense to reflect this in our docs somehow.